lxd/images.go

```diff
@@ -173,7 +173,7 @@ func imgPostContInfo(d *Daemon, r *http.Request, req api.ImagesPost, op *operati
 		if !shared.IsSnapshot(name) {
 			return nil, fmt.Errorf("Not a snapshot")
 		}
-	case "container":
+	case "container", "virtual-machine", "instance":
```

Comment:
@stgraber should we validate the ctype specified with the instance type loaded on line 192?

Reply:
No as our current client isn't very smart and always passes container :)

Reply:
I think I'll send a follow-up fix to have the client always pass "instance" so long as we have the instances API extension, will make things a bit less wrong there :)

Reply:
(for speed reasons, the client doesn't even pull the instance so it doesn't know what the type is)

Reply:
@stgraber ok makes sense, and that sounds good. Thanks